crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -164,8 +165,8 @@ fn validate_override_with_default_configs(
             let verrors = e.collect::<Vec<ValidationError>>();
             log::error!("({key}) config key validation error: {:?}", verrors);
             return Err(validation_error!(
-                "schema validation failed for {key} with error {:?}",
-                verrors
+                "schema validation failed for {key}: {}",
+                validation_err_to_str(verrors).first().unwrap()
```

Comment:
we can do `unwrap_or(verrors)` as a fallback.

Reply:
Not needed I guess, there will be always be one item in the vector

Reply:
Anyway its a str, i will change this to `unwrap_or(String::new())`